rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,111 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+Versioned Experimental Data Structure Conceptual Model
```

Comment:
Please replace _RFC Name_ with _Versioned Experimental Data Structure Conceptual Model_ which is the name of this RFC. The instructions in the template - _Replace "RFC Name" above with the name of your RFC. Keep it short and descriptive._